src/Microsoft.ML.Core/Data/ModelLoading.cs

```diff
@@ -259,7 +276,7 @@ private void EnsureLoaderAssemblyIsRegistered(ComponentCatalog catalog)
         {
             if (!string.IsNullOrEmpty(LoaderAssemblyName))
```

Comment:
>LoaderAssemblyName [](start = 38, length = 18)

It may have been more sensible to do `ForwardedLoaderAssemblyName` here, but we can fix that later since the scenarios that would require that change can handle that change independently.